<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TextElemDlg</class>
 <widget class="QDialog" name="TextElemDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Text Element</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Reference Corner:</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="rightbottom_radiobtn" >
        <property name="text" >
         <string>R&amp;ight Bottom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="leftbottom_radiobtn" >
        <property name="text" >
         <string>L&amp;eft Bottom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="righttop_radiobtn" >
        <property name="text" >
         <string>&amp;Right Top</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="lefttop_radiobtn" >
        <property name="text" >
         <string>&amp;Left Top</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QCheckBox" name="absolute_placing_checkbox" >
     <property name="toolTip" >
      <string>Absolute placed items are fixed to a screen position, relative placed items are fixed to a LAT/LON point.</string>
     </property>
     <property name="text" >
      <string>&amp;Place absolute (see tooltip)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="draw_border_checkbox" >
     <property name="text" >
      <string>&amp;Draw borders around the text</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Text:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="text_edit" >
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>text_edit</tabstop>
  <tabstop>draw_border_checkbox</tabstop>
  <tabstop>absolute_placing_checkbox</tabstop>
  <tabstop>lefttop_radiobtn</tabstop>
  <tabstop>righttop_radiobtn</tabstop>
  <tabstop>leftbottom_radiobtn</tabstop>
  <tabstop>rightbottom_radiobtn</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TextElemDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>358</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TextElemDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>441</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>absolute_placing_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>384</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel" >
     <x>376</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
